src/main/java/seedu/address/logic/parser/ArgumentMultimap.java

```diff
@@ -84,6 +84,7 @@ public String getValueOrEmpty(Prefix prefix) {
 
     /**
      * Checks if {@link #argMultimap} is empty.
+     * @return The boolean result.
```

Comment:
The @return could be more descriptive. E.g. `@return true if empty; otherwise false`.